utility/environment_check.py

```diff
@@ -0,0 +1,127 @@
+"""
+Util for environment check befora and after test to compare and find stale
+leftovers
+"""
+import logging
+import pytest
+
+from ocs import ocp, defaults, constants, exceptions
+from deepdiff import DeepDiff
+
+log = logging.getLogger(__name__)
+
+
+POD = ocp.OCP(kind=constants.POD)
+SC = ocp.OCP(kind=constants.STORAGECLASS)
+CEPHFILESYSTEM = ocp.OCP(kind=constants.CEPHFILESYSTEM)
+CEPHBLOCKPOOL = ocp.OCP(kind=constants.CEPHBLOCKPOOL)
+PV = ocp.OCP(kind=constants.PV)
+PVC = ocp.OCP(kind=constants.PVC)
+SECRET = ocp.OCP(kind=constants.SECRET)
+NS = ocp.OCP(kind=constants.NAMESPACE)
+
+ADDED_RESOURCE = 'iterable_item_added'
+REMOVED_RESOURCE = 'iterable_item_removed'
+
+
+@pytest.fixture(scope='class')
+def environment_checker(request):
+    request.addfinalizer(get_status_after_execution)
+    get_status_before_execution()
+
+
+def get_status_before_execution():
```

Comment:
Shouldn't we fetch this data out of the CephCluster yaml?
This will ensure we're comparing against the desired state, and will apply also for deploymet - we'll be able to execute this health check after the deployment is done

Reply:
As discussed f2f, there are a lot of implications by fetching that info from cephcluster yaml.
As we need a quick solution that provides us with what we need, this PR answers that